src/popup/router/pages/SignTransaction.vue

```diff
@@ -0,0 +1,67 @@
+<template>
+  <!-- eslint-disable vue-i18n/no-raw-text -->
+  <div class="sign-transaction">
+    <div class="section-title">Sign transaction for</div>
+
+    <div class="url-bar link-sm text-left">
+      {{ callbackOrigin }}
+    </div>
+
+    <div class="section-title">Transaction</div>
+
+    <div class="tip-note-preview mt-15">
+      {{ $route.query.encodedTx }}
+    </div>
+
+    <Button @click="signTransaction" :disabled="disabled">
+      {{ $t('pages.tipPage.confirm') }}
+    </Button>
+    <Button @click="openCallbackOrGoHome(false)">
+      {{ $t('pages.tipPage.cancel') }}
+    </Button>
+  </div>
+</template>
+
+<script>
+import { mapGetters } from 'vuex';
+import Button from '../components/Button';
+import deeplinkApi from '../../../mixins/deeplinkApi';
+
+export default {
+  components: { Button },
+  mixins: [deeplinkApi],
+  data: () => ({ disabled: false }),
+  computed: mapGetters(['account']),
+  methods: {
+    async signTransaction() {
+      await this.$watchUntilTruly(() => this.$store.state.sdk);
+      const signedTransaction = await this.$store.state.sdk.signTransaction(
+        this.$route.query.encodedTx,
+        this.account.address,
+      );
+      this.disabled = true;
+      await this.$store.state.sdk.sendTransaction(signedTransaction, this.account.address);
```

Comment:
I think sending should be done by aepp. But it may be easier to send transaction from here, I'm proposing can add a `broadcast` option, disabled by default.

Reply:
The second argument of `sendTransaction` is also object